### PR TITLE
Feature/ar 548 GitHub actions

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: src/Api/Fixtures/MediaFixtures.php [hunk 1]
```diff
@@ -62,15 +62,15 @@ public function __construct()
...
+     * Get media.
      *
      * @param $id
-     *   Media ID to fetch.
+     *   Media ID to fetch
      *
-     * @return Media|null
-     *   Media object or null if not found.
+     * @return media|null
+     *                    Media object or null if not found
```
Comment (tuj) [inline src/Api/Fixtures/MediaFixtures.php:71]: Do we want this rule?

**Review by tuj — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+on: pull_request
+name: Review
+jobs:
+  test-composer-install:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.4']
```
Comment (cableman) [inline .github/workflows/pr.yaml:9]: Should we not add php 8.0 here?

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Yarn install
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: yarn install
+      - name: markdownlint
+        run: yarn markdownlint README.md docs
```
Comment (cableman) [inline .github/workflows/pr.yaml:89]: Newline

File: composer.json [hunk 4]
```diff
@@ -8,11 +10,12 @@
         "ext-ctype": "*",
         "ext-iconv": "*",
         "api-platform/core": "^2.6",
-        "composer/package-versions-deprecated": "1.11.99.2",
+        "composer/package-versions-deprecated": "^1.11",
         "doctrine/annotations": "^1.0",
         "doctrine/doctrine-bundle": "^2.4",
         "doctrine/doctrine-migrations-bundle": "^3.1",
         "doctrine/orm": "^2.9",
+        "friendsofphp/php-cs-fixer": "^3.0",
```
Comment (cableman) [inline composer.json:18]: Should this be dev requirement

File: package.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "license": "UNLICENSED",
```
Comment (cableman) [inline package.json:2]: Missing: name an description

File: src/Api/Fixtures/MediaFixtures.php (hunk 1 above)
Comment (cableman) [reply src/Api/Fixtures/MediaFixtures.php:71]: NO

**Review by cableman — CHANGES_REQUESTED** [review]

File: src/Api/Fixtures/MediaFixtures.php (hunk 1 above)
Comment (turegjorup) [reply src/Api/Fixtures/MediaFixtures.php:71]: I don't like it but I also don't want to deviate from https://symfony.com/doc/current/contributing/code/standards.html

**Comment by turegjorup** [comment]
@cableman We don't deviate from standards. Even if don't like them: https://symfony.com/doc/current/contributing/code/standards.html#making-your-code-follow-the-coding-standards

File: src/Api/Fixtures/MediaFixtures.php (hunk 1 above)
Comment (tuj) [reply src/Api/Fixtures/MediaFixtures.php:71]: Then it should be on a single line

Comment (turegjorup) [reply src/Api/Fixtures/MediaFixtures.php]: Done

**Pull request merged**